packages/lit-element/src/test/node-imports.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// This file will be loaded by Node from the node:test script to verify that all
+// exports of this package can be imported without crashing in Node.
+
+import 'lit-element';
+import 'lit-element/experimental-hydrate-support.js';
+import 'lit-element/private-ssr-support.js';
+import 'lit-element/decorators.js';
+import 'lit-element/decorators/custom-element.js';
+import 'lit-element/decorators/event-options.js';
+import 'lit-element/decorators/state.js';
+import 'lit-element/decorators/property.js';
+import 'lit-element/decorators/query.js';
+import 'lit-element/decorators/query-all.js';
+import 'lit-element/decorators/query-assigned-elements.js';
+import 'lit-element/decorators/query-assigned-nodes.js';
+import 'lit-element/decorators/query-async.js';
+
+import {LitElement, html} from 'lit-element';
+import {customElement} from 'lit-element/decorators.js';
+
+@customElement('my-element')
```

Comment:
Make sense to include a version that doesn't use the decorator as well?

Reply:
Done